shared/tests/utility/math/docs/README.txt

```diff
@@ -0,0 +1,14 @@
+The files in this folder contain the same input data used in TestCoordinates.cpp along with some matlab scripts that perform the same calculations found in coordinates.h
+The output from matlab is used to validate the coordinates.h calculations in the NUBOTS codebase.
+Test input data can be found in both matlab and c++ formats. The matlab format data is passed into the functions starting with calc_****.m to produce the output data.
+Test output data provided is in a cpp format.
+
+NOTE: During initial testing when values at the extremes of the data types domain, DBL_MAX and DBL_MIN, were found to cause inf return values when passed to the functions
+being tested. While normally some measures would have been put in place to handle this, it was determined that the performance impact would be to great, and that these values won't be
+possible during normal operation.
+The maximum test values that were determined to be safe are:
+TEST_DBL_MAX    = 5.9667e+153;
+TEST_DBL_MIN    = 1.4853e-307;
+
+The values returned compared to the pre-calculated values should be within the error threshold:
+ERROR_THRESHOLD = 1e-6;
```

Comment:
Didn't this change to being a percentage threshold, rather than an absolute threshold?

Reply:
Yep. That change happened further down where instead of just comparing the difference between the pre-calc value and the method result in the REQUIRE statement, i used Approx(cart_compare.x()).epsilon(ERROR_THRESHOLD).

Reply:
I think this comment in this readme could reflect that we have opted to use a percentage threshold (and also even any reasoning behind that threshold or why we used percentages, if you think that could be useful to future maintainers?)

Reply:
Read the information about https://en.cppreference.com/w/cpp/types/numeric_limits/epsilon which will help you write about how you should be testing. It shouldn't be a percentage error or an absolute error per say. It should be an error based on the machine epsilon (which is based on the relative magnitude of the numbers in the operation)